File: src/main/scala/top/scaleda/systemverilog/psi/nodes/ScopeNode.java
package top.scaleda.systemverilog.psi.nodes;

import com.intellij.psi.PsiElement;
import com.intellij.psi.PsiNamedElement;

import java.util.Collection;

public interface ScopeNode extends PsiElement {

    Collection<PsiNamedElement> getAvailableNamedElements();

}

File: src/main/scala/top/scaleda/bluespec/psi/nodes/ScopeNode.scala
package top.scaleda.bluespec.psi.nodes

import com.intellij.psi.PsiElement
import com.intellij.psi.PsiNamedElement
import java.util

trait ScopeNode extends PsiElement { def getAvailableNamedElements: util.Collection[PsiNamedElement] }
